### PR TITLE
Add FFMpegCore.Extensions.SkiaSharp

### PR DESCRIPTION
**Pull request by drasive**
Adds a new project and corresponding NuGet package FFMpegCore.Extensions.SkiaSharp. It is based on FFMpegCore.Extensions.SystemDrawingCommon but uses SkiaSharp instead of SystemDrawingCommon, the former being supported on Windows when using .NET 7. See also https://github.com/rosenbjerg/FFMpegCore/issues/381.

FFMpegCore.Test was adjusted so that the tests contibue to be executed for Extensions.SystemDrawingCommon as-is but are also executed for the new Extensions.SkiaSharp implementation.

I have little familiarity with SkiaSharp and cannot vouch for the correctness of the implementation. If it does not pass the bar for being merged, it may still be useful to some or as a starting point.
Areas that could be improved:
* There could be more tests that check if the output of Extensions.SkiaSharp looks visually correct.
* FFMpegCore.Extensions.SkiaSharp duplicates a lot of code from FFMpegCore.Extensions.SystemDrawingCommon.
* FFMpegCore.Test is now more complex because it tests both implementations.

I may publish this version of FFMpegCore.Extensions.SkiaSharp to NuGet under the package name "**DimitriVranken.**FFMpegCore.Extensions.SkiaSharp" as not to cause a conflict with a potential later "official" release called "FFMpegCore.Extensions.SkiaSharp".

### CODE REVIEW

**Comment by rosenbjerg** [comment]
Hey Dimitri, looks very interesting :)
If it has enough test coverage and passes CI, I don't see a reason not to merge it

File: FFMpegCore.Examples/Program.cs [hunk 1]
```diff
@@ -79,7 +79,8 @@ await FFMpegArguments
     FFMpeg.PosterWithAudio(inputPath, inputAudioPath, outputPath);
     // or 
 #pragma warning disable CA1416
-    using var image = Image.FromFile(inputImagePath);
+    using var image = Image.FromFile(inputImagePath);    // Using FFMpegCore.Extensions.System.Drawing.Common
+    //using var image = SKBitmap.Decode(inputImagePath); // Using FFMpegCore.Extensions.SkiaSharp
```
Comment (rosenbjerg) [inline FFMpegCore.Examples/Program.cs]: Uncomment and rename to skiaImage or similar? Please avoid commented out code :)
You could also add a second block for the skia example, to avoid confusion

File: FFMpegCore.Extensions.SkiaSharp/FFMpegCore.Extensions.SkiaSharp.csproj [hunk 2]
```diff
@@ -0,0 +1,22 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+    <PropertyGroup>
+        <IsPackable>true</IsPackable>
+        <Description>Image extension for FFMpegCore using System.Common.Drawing</Description>
```
Comment (rosenbjerg) [inline FFMpegCore.Extensions.SkiaSharp/FFMpegCore.Extensions.SkiaSharp.csproj]: Replace `System.Common.Drawing` with `SkiaSharp`

File: FFMpegCore.Extensions.SkiaSharp/FFMpegCore.Extensions.SkiaSharp.csproj [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+        <PackageVersion>5.0.0</PackageVersion>
+        <PackageReleaseNotes>
+        </PackageReleaseNotes>
+        <PackageTags>ffmpeg ffprobe convert video audio mediafile resize analyze muxing</PackageTags>
+        <Authors>Malte Rosenbjerg, Vlad Jerca, Max Bagryantsev</Authors>
```
Comment (rosenbjerg) [inline FFMpegCore.Extensions.SkiaSharp/FFMpegCore.Extensions.SkiaSharp.csproj]: Replace names here with your name, or add your name to the list

File: FFMpegCore.Extensions.SkiaSharp/FFMpegCore.Extensions.SkiaSharp.csproj [hunk 4]
```diff
@@ -0,0 +1,22 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+    <PropertyGroup>
+        <IsPackable>true</IsPackable>
+        <Description>Image extension for FFMpegCore using System.Common.Drawing</Description>
+        <PackageVersion>5.0.0</PackageVersion>
+        <PackageReleaseNotes>
+        </PackageReleaseNotes>
+        <PackageTags>ffmpeg ffprobe convert video audio mediafile resize analyze muxing</PackageTags>
```
Comment (rosenbjerg) [inline FFMpegCore.Extensions.SkiaSharp/FFMpegCore.Extensions.SkiaSharp.csproj]: Adjust tags, e.g. mention skiasharp or similar

File: testenvironments.json [hunk 5]
```diff
@@ -0,0 +1,10 @@
+{
```
Comment (rosenbjerg) [inline testenvironments.json:1]: Is this file needed?

**Comment by rosenbjerg** [comment]
I cannot push changes to your PR btw, although I could resolve the conflicts

File: testenvironments.json (hunk 5 above)
Comment (drasive) [reply testenvironments.json:1]: The file allows execution of unit test directly from Visual Studion in WSL when developing on Windows, see the documentation here: https://learn.microsoft.com/en-us/visualstudio/test/remote-testing?view=vs-2022

**Comment by drasive** [comment]
I have verified the visual output with some of my actual uses cases, in addition to the unit tests, and everything seems to work fine. The Codecov checks fail but the code coverage seems fine to me and is on par with the existing code.

File: testenvironments.json (hunk 5 above)
Comment (rosenbjerg) [reply testenvironments.json:1]: Alright

**Comment by rosenbjerg** [comment]
Looks great @drasive 
Thanks for the contribution to the project
